platzky_redirections/plugin.py

```
@@ -93,14 +67,32 @@ def setup_routes(app, redirections):
     named_redirect.__name__ = name
     return named_redirect
 
 
-def process(app: Engine, config: dict[str, str]) -> Engine:
-    redirections = parse_redirections(config)
-    setup_routes(app, redirections)
-    return app
+class RedirectionsConfig(PluginBaseConfig):
+    """Configuration model for the redirections plugin."""
+
+    pass
```

Ruff (0.15.0)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)